modules/identity/src/Volo.Abp.Identity.EntityFrameworkCore/Volo/Abp/Identity/EntityFrameworkCore/IIdentityDbContext.cs

```diff
@@ -17,5 +17,7 @@ public interface IIdentityDbContext : IEfCoreDbContext
         DbSet<OrganizationUnit> OrganizationUnits { get; set; }
 
         DbSet<IdentitySecurityLog> IdentitySecurityLogs { get; set; }
+
+        DbSet<IdentityLinkUser> IdentityLinkUsers { get; set; }
```

Comment:
`LinkUsers` would be enough for the property name. Same for `SecurityLogs` too (take Users, Roles.. as examples)